tests/run/importfrom.pyx

```diff
@@ -68,6 +68,10 @@ def typed_imports():
     try:
         from sys import version_info as maxunicode
     except TypeError, e:
+        if '__pypy__' in sys.builtin_module_names:
```

Comment:
This would actually be a _good_ use case for `getattr()`, because it's obvious.
```suggestion
        if getattr(sys, "pypy_version_info", None):
```